public/app/features/search/page/components/SearchView.tsx

```diff
@@ -93,19 +94,40 @@ export const SearchView = ({
   // Search usage reporting
   useDebounce(
```

Comment:
not related to the change, but maybe we can also cancel the debounce when unmounting the page? Im thinking it  could affect the reporting of the events when the user immediately move to other pages?

Reply:
Good catch! I thought about it, but in reality, the event must be sent to not break the funnels. I mean, if you have the user going to another page in less than 1 second, you will lose track of this view.
For example: Home -> Search -> Dashboard X; if we cancel de debounce, the funnel will be Home -> Dashboard X; which is not real.